src/technologies/a.json

```diff
@@ -2454,21 +2454,53 @@
     "scriptSrc": "cf\\.aliyun\\.com/",
     "website": "https://help.aliyun.com/document_detail/193141.html"
   },
-  "All in One SEO Pack": {
+  "All in One SEO": {
     "cats": [
       54,
       87
     ],
     "cpe": "cpe:2.3:a:aioseo:all_in_one_seo:*:*:*:*:*:wordpress:*:*",
-    "description": "All in One SEO plugin optimizes WordPress website and its content for search engines.",
-    "html": "<!-- All in One SEO Pack ([\\d.]+) \\;version:\\1",
+    "description": "All in One SEO optimizes a WordPress website and its content for search engines.",
+    "meta": {
+      "generator": "^All in One SEO \\(AIOSEO\\) ([\\d\\.]+)\\;version:\\1"
+    },
+    "dom": {
+      "script.aioseo-schema": {
+        "attributes": {
+          "class": ""
```

Comment:
Do we need these lines? And if it has `aioseo-schema` class then why is the class attribute empty?

Reply:
I honestly don't know, I copied this from Yoast SEO's detection :)